### PR TITLE
LPS-175831 Don't allow focusing the tabItem using mouse, it should only be possible using the keyboard

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:forward

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:stop

**Comment by p2kmgcl** [comment]
horrible ci:close

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments_and_widgets/hooks/useKeyboardNavigation.js [hunk 1]
```diff
@@ -118,6 +121,7 @@ function onHeaderKeyDown(element, keyCode, handleOpen) {
 		if (previousList) {
 			const lastItem = previousList.lastChild;
 
+			lastItem.setAttribute('tabindex', -1);
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/fragments_and_widgets/hooks/useKeyboardNavigation.js:124]: I am not sure if we should be updating HTML attributes of elements that are being managed by React. Let me think about this a little bit and see if we can came up with another solution :thinking:

**Pull request closed without merging**